### DISCUSSION TITLE
GraphQL noob question

### DISCUSSION THREAD

**Discussion by LenaNoctua** [git-bug/git-bug#589] [General]
Sorry for this noob question, but how can I refresh those *.generated.tsx-files, if I added some fields in a *.graphql-file on my branch?
@MichaelMure @claudioantonio 

I use IntelliJ.

Thanks in advance!

**Comment by MichaelMure** [comment]
I don't know for sure as I didn't write that part but I believe you can just run the `generate` target in `package.json`: https://github.com/MichaelMure/git-bug/blob/master/webui/package.json#L44-L48

Note that this is also executed when you run for example `npm start`.

**Reply by LenaNoctua** [reply]
Thanks a lot for the quick answer! It took me some minutes to find out to run it in our docker container, but now "npm start" worked :)

**Comment by MichaelMure** [comment]
Also, have a look at the Readme for the webUI: https://github.com/MichaelMure/git-bug/blob/master/webui/Readme.md
